cg/constants/taxprofiler.py

```diff
@@ -0,0 +1,22 @@
+"""Taxprofiler related constants."""
+
+
+from cg.constants.nextflow import NFX_SAMPLE_HEADER, NFX_SAMPLESHEET_READS_HEADERS
+
+TAXPROFILER_RUN_ACCESSION = "run_accession"
+TAXPROFILER_INSTRUMENT_PLATFORM = "instrument_platform"
+TAXPROFILER_ACCEPTED_PLATFORMS = ["ILLUMINA", "OXFORD_NANOPORE"]
+TAXPROFILER_FASTA_HEADER = "fasta"
+TAXPROFILER_SAMPLESHEET_HEADERS = [
+    NFX_SAMPLE_HEADER,
+    TAXPROFILER_RUN_ACCESSION,
+    TAXPROFILER_INSTRUMENT_PLATFORM,
+]
+TAXPROFILER_SAMPLESHEET_HEADERS.extend(NFX_SAMPLESHEET_READS_HEADERS)
+TAXPROFILER_SAMPLESHEET_HEADERS.append(TAXPROFILER_FASTA_HEADER)
```

Comment:
The reason I am having it like that is because the headers of the samplesheet have to be as:

`sample,run_accession,instrument_platform,fastq_1,fastq_2,fasta`

I have not tested your suggestion but I am thinking that might reorder the columns?